### DISCUSSION TITLE
Watch in Plex?

### DISCUSSION THREAD

**Discussion by devanteweary** [seerr-team/seerr#595] [Q&A]
Hey guys, amazing app. Literally contributed to me getting off my butt and setting up a NAS for the first time in my life.

My question is... you know how there's "Watch in Jellyfin"?
Can we set up a button to go to Plex instead?

99% of my friends/family use Plex so just wondering.

Thank you!

**Comment by fallenbagel** [comment]
Play in plex comes up when you setup jellyseerr with plex. If you setup jellyseerr with jellyfin you get play in jellyfin button. Similarly emby.

Simultaneous multiple server management is a planned and coming soon feature.

**Reply by devanteweary** [reply]
You know what? I looked around for somewhere to set up Plex intergration but only see a Jellyfin tab in settings.

I know I'm probably being obtuse and missing it!
Where would I go to point it to Plex?

Using the Unraid > Docker container by the way!

**Reply by fallenbagel** [reply]
> You know what? I looked around for somewhere to set up Plex intergration but only see a Jellyfin tab in settings.
> 
> I know I'm probably being obtuse and missing it!
> Where would I go to point it to Plex?
> 
> Using the Unraid > Docker container by the way!

Like I said earlier. You can only setup a single server. Simultaneous server setup is a planned feature. 

You'll have to start over and login from plex to use plex or login from jellyfin to use jellyfin

**Reply by devanteweary** [reply]
OK I get it now. After reading more documentation and your reply, it looks like you decided between Jellyfin and Plex upon that initial Jellyseerr login.

Well I finally got everything set up perfectly (loving the SMTP notifications to my friends/family and Discord for myself) so I'll wait until the day!
